### PR TITLE
SWMAAS-1448 Walk Time UI Tweaks

### PR DESCRIPTION
**Pull request by dtran-phunware**
###### Context
PR for making UI Tweaks requested by the Design. Since Design does not have github accounts, I have shared screenshots with Design (Suzy) who has approved.

###### Description
* Converted Walk Time ViewGroup into ConstraintLayout to aid in vertical centering of elements in box.
* Converted entire layout to be wrapped by a coordinator layout to leverage bottom sheet behavior for shadows.

image

### CODE REVIEW

File: Samples/kotlin/src/main/res/layout/activity_routing.xml [hunk 1]
```diff
@@ -1,165 +1,186 @@
...
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <android.support.constraint.ConstraintLayout
+            android:id="@+id/walk_time_view"
+            android:layout_width="match_parent"
+            android:layout_height="110dp"
+            android:background="@color/white"
+            android:elevation="12dp"
+            android:outlineProvider="paddedBounds"
```
Comment (npike) [inline Samples/kotlin/src/main/res/layout/activity_routing.xml]: I can't recall from your screen share if "paddedBounds" had any effect over not providing this attribute at all.  If that's the case, can we remove the attribute?

Comment (dtran-phunware) [reply Samples/kotlin/src/main/res/layout/activity_routing.xml]: darnit that was left in on accident

Comment (dtran-phunware) [reply Samples/kotlin/src/main/res/layout/activity_routing.xml]: actually nick i think it does work. paddedBound does give us the shadow

Comment (dtran-phunware) [reply Samples/kotlin/src/main/res/layout/activity_routing.xml]: nvm you were right.

**Pull request merged**